### PR TITLE
Increase penalty box size to prevent keeper getting stuck

### PR DESCRIPTION
**Pull request by MaikRe**
## Introduced Changes

Increase the penalty box size by 20cm and shift accordingly
The keeper was getting stuck on the boundary or thinking it was outside the penalty box and would not defend a penalty kick properly.

Fixes #

## How to Test

Optimally merge #811 first.
It can be seen that the keeper now sticks to the line when defending the penalty kick rather than staying behind it and failing to defend properly.

### CODE REVIEW

File: crates/control/src/rule_obstacle_composer.rs [hunk 1]
```diff
@@ -122,11 +122,11 @@ pub fn create_penalty_box(field_dimensions: &FieldDimensions, kicking_team: Team
     };
     let half_field_length = field_dimensions.length / 2.0;
     let half_penalty_area_length = field_dimensions.penalty_area_length / 2.0;
-    let center_x = side_factor * (half_field_length - half_penalty_area_length);
+    let center_x = side_factor * (half_field_length - half_penalty_area_length + 0.1);
     RuleObstacle::Rectangle(Rectangle::new_with_center_and_size(
         point![center_x, 0.0],
         vector![
-            field_dimensions.penalty_area_length,
+            field_dimensions.penalty_area_length + 0.2,
```
Comment (schluis) [inline crates/control/src/rule_obstacle_composer.rs:129]: Please convert these to modifiable parameters ;)

**Review by schluis — CHANGES_REQUESTED** [review]

File: crates/control/src/rule_obstacle_composer.rs (hunk 1 above)
Comment (MaikRe) [reply crates/control/src/rule_obstacle_composer.rs]: Did so, and I grouped the rule obstacle relevant parameters into one part of the ```default.json```. Do you think I should make a struct to group these further?

Comment (schluis) [reply crates/control/src/rule_obstacle_composer.rs]: Nice, is fine for me like this 👍🏻

File: etc/parameters/default.json [hunk 2]
```diff
@@ -1160,6 +1160,9 @@
     }
   },
   "angular_velocity_smoothing_factor": 0.1,
-  "center_circle_obstacle_increase": 1.2,
-  "free_kick_obstacle_radius": 0.75
+  "rule_obstacles": {
+    "center_circle_obstacle_increase": 1.2,
+    "free_kick_obstacle_radius": 0.75,
+    "penaltykick_box_increase": 0.2
```
Comment (schluis) [inline etc/parameters/default.json:1166]: Please make both `increase`s consistent, the `center_circle_obstacle_increase` is multiplicative and the new `penaltykick_box_increase` is additive.

Comment (MaikRe) [reply etc/parameters/default.json]: Renamed

**Review by schluis — APPROVED** [review]

**Comment by schluis** [comment]
Nice, thanks! 👍🏻

**Pull request merged**